app/test/integration-test/pom.xml

```diff
@@ -36,13 +36,14 @@
 
     <syndesis.integration.runtime>spring-boot</syndesis.integration.runtime>
     <syndesis.camel.k.customizers>health,logging,syndesis</syndesis.camel.k.customizers>
-    <syndesis.logging.enabled>false</syndesis.logging.enabled>
+    <syndesis.logging.enabled>true</syndesis.logging.enabled>
     <syndesis.server.port>8080</syndesis.server.port>
     <syndesis.management.port>8081</syndesis.management.port>
     <syndesis.debug.port>5005</syndesis.debug.port>
     <syndesis.debug.enabled>false</syndesis.debug.enabled>
     <syndesis.s2i.build.enabled>false</syndesis.s2i.build.enabled>
     <syndesis.output.directory>${project.build.directory}/integrations</syndesis.output.directory>
+    <syndesis.project.mount.path>/tmp/src</syndesis.project.mount.path>
```

Comment:
Are we assuming linux/unix here? Is that fine? No Windows?

Reply:
windows? what's that? 😄 just kidding. good point. in Maven this will be difficult to set according to os but I think we can automatically take care of proper paths in Java code. Let me fix this

Reply:
while trying to fix I realized that this is a mount path inside of the Docker container. And this will always be a unix style environment. so no need to support windows here.